cs.py

```diff
@@ -99,7 +99,7 @@ def main():
 
     if args.number > 0 and args.wipe == False:
         from modules import retainnumberofreports
-        retainnumberofreports.retainReports(args.environment, int(args.number))
+        retainnumberofreports.retain_reports(args.environment, int(args.number))
```

Comment:
if we are following up some standard then this retainnumberofreports , should be like this retain_number_of_reports